index.js

```diff
@@ -40,41 +42,39 @@ var cssLintPlugin = function(options) {
 
   var ruleset = {};
 
-  // Read CSSLint options from a specified csslintrc file.
-  if (typeof options === 'string') {
-    // Don't catch readFile errors, let them bubble up
-    var externalOptions = fs.readFileSync('./'+options);
-
-    try {
-      options = JSON.parse(externalOptions);
-    }
-    catch(err) {
-      throw new Error('Error parsing csslintrc: '+err);
-    }
-  }
+  var rcLoader = new RcLoader('.csslintrc', options, { loader: 'async' });
 
   // Build a list of all available rules
   csslint.getRules().forEach(function(rule) {
     ruleset[rule.id] = 1;
   });
 
-  for (var rule in options) {
-    if (!options[rule]) {
-      // Remove rules that are turned off
-      delete ruleset[rule];
-    }
-    else {
-      ruleset[rule] = options[rule];
-    }
-  }
-
```

Comment:
Looks like we lose support for [passing a rules configuration to the plugin as documented here](https://github.com/lazd/gulp-csslint#csslintruleconfiguration).